src/components/data-table/table-batch-actions.ts

```diff
@@ -47,6 +47,12 @@ class BXTableBatchActions extends LitElement {
   @property({ type: Number, attribute: 'selected-rows-count' })
   selectedRowsCount = 0;
 
+  updated(changedProperties) {
+    if (changedProperties.has('active')) {
+      this.setAttribute('tabindex', `${this.active ? 0 : -1}`);
```

Comment:
The reason why I was against adding `tabindex` to the host element is because we don't want the host element itself to be focused - We want one of the buttons, etc. in the Shadow DOM to be focused instead. Seems that making the attribute value empty works for that purpose.

```suggestion
      this.setAttribute('tabindex', `${this.active ? '' : '-1'}`);
```